group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -3125,25 +3128,25 @@ public void testSessionTimeoutExpiration() {
         List<ExpiredTimeout<Void, CoordinatorRecord>> timeouts = context.sleep(45000 + 1);
 
         // Verify the expired timeout.
+        assertEquals(1, timeouts.size());
```

Comment:
Is there a reason why we need to change this code? Ah, it may be because of the append future. Is it the case?